src/auth/api/http/auth.go

```diff
@@ -29,8 +30,8 @@ func (m *Auth) Middleware(next http.Handler) http.Handler {
 		ctx := r.Context()
 
 		// If no auth method, return wildcard user
-		if m.authenticator == nil {
-			next.ServeHTTP(rw, r.Clone(WithUserInfo(ctx, entities.NewWildcardUser())))
+		if reflect.ValueOf(m.authenticator).IsNil() {
```

Comment:
Please no usage of reflect in the code, if you need to use reflect then this solution is not the correct one.

Reply:
Since we expect an interface here we cannot run the `== nil` here, do you have a different suggestion than reflect?

Reply:
Like it was implemented before, 2 middlewares, one `Auth` and one `Wildcard` like I had implemented before.

Reply:
Oki, I will do that